pkg/kubelet/prober/worker.go

```diff
@@ -105,13 +108,14 @@ func (w *worker) run() {
 		w.probeManager.removeWorker(w.pod.UID, w.container.Name, w.probeType)
 	}()
 
+	tickPeriod := time.Duration(w.spec.PeriodSeconds) * time.Second
 probeLoop:
 	for w.doProbe() {
 		// Wait for next probe tick.
 		select {
 		case <-w.stop:
 			break probeLoop
-		case <-probeTicker.C:
+		case <-time.Tick(wait.Jitter(tickPeriod, workerProbeJitterFactor)):
```

Comment:
Correct, the ticker needs to be captured outside the loop. time.After has slightly different semantics - it will have a fixed amount of time between each `doProbe()` call, where as a ticker will have a fixed amount of time for the total loop (including teh `doProbe()` time). `time.After` may be preferred in this case, as it will essentially "re-jitter" each loop, so a bad roll of the jitter-dice won't have lasting effects.